app/jobs/background_render_job.rb

```diff
@@ -3,6 +3,15 @@ class BackgroundRenderJob < BaseJob
   queue_as :short_running
 
   def perform(render_id, **options)
+    # Before we run this, let's make sure someone still wants it.
+    pubsub = ActionCable.server.pubsub
+    channels = pubsub.send(:redis_connection).pubsub('channels', '*')
+    # `channels` is an array of colon-separated strings. One string will contain a uuid.
+    # We don't know the order of the elements, but we just need the uuid.
+    uuid_regex = /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/
+    uuid = channels.join(':').split(':').filter { |i| uuid_regex.match(i.downcase) }.first
```

Comment:
I think this might be better written as 

```
dev = Rails.environment.dev?
found_stream = channels.detect { |stream| stream.include?(render_id) }
return unless dev || found_stream
```

Reply:
Nice to know about the detect method. Thanks. I'm curious about why we want dev to behave differently.

Reply:
Okay. I was having issues with my local installation, but with some restarts I think that we can probably do without the differentiation for dev.